data/json/techniques.json

```
@@ -260,6 +260,72 @@
     "defensive": true,
     "grab_break": true
   },
+  {
+    "type": "technique",
+    "id": "tec_bojutsu_thrust",
+    "name": "Front Thrust",
+    "messages": [ "You thrust at the %s", "<npcname> thrusts at %s" ],
+    "skill_requirements": [ { "name": "melee", "level": 1 } ],
+    "melee_allowed": true,
+    "mult_bonuses": [ { "stat": "damage", "type": "bash", "scale": 1.2 } ]
+  },
+  {
+    "type": "technique",
+    "id": "tec_bojutsu_kneestrike",
+    "name": "Knee Strike",
+    "messages": [ "After blocking %s's attack you quickly strike at their knee!" ],
```

```suggestion
    "messages": [ "After blocking %s's attack you quickly strike at their knee!", "After blocking %s's attack <npcname> quickly strikes at their knee!" ],
```
Need an NPC version, i suppose